### PR TITLE
Page structure

### PR DESCRIPTION
**Pull request by CollinsTatang**
- Use Gitflow to management the project on Github.
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling.
- Use ES6 modules, with import and export.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "webpack-project",
+    "version": "1.0.0",
+    "description": "",
+    "private": true,
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "webpack serve --open",
+        "build": "webpack"
```
Comment (vichuge) [inline package.json]: - [x] On this part please add `"dev": "webpack --mode development",`. This is to run the project in development mode. 🐶

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="h4 font-weight-bold">Add your score</div>
+        <form action="#" class="d-flex form">
+          <input type="text" class="name form-control mb-4" placeholder="Enter name">
+          <input type="number" class="score form-control mb-4" placeholder="Enter score">
+          <input type="submit" value="submit" id="submit" class="btn btn-success btn-sm">
+        </form>
+      </div>
+    </div>
+    </div>
+  </div>
```
Comment (vichuge) [inline src/index.html]: - [x] I kindly invite you to build all this in the `index.js` file. As a good practice, the index.html main contain only the root element (index.js in this case). 🐱
- Don't forget to use all ES6 best practices for this.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @CollinsTatang 👋🏾 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**